core/server/proxy/src/main/java/alluxio/web/ProxyWebServer.java

```diff
@@ -39,12 +38,9 @@ public final class ProxyWebServer extends WebServer {
   public static final String ALLUXIO_PROXY_SERVLET_RESOURCE_KEY = "Alluxio Proxy";
   public static final String FILE_SYSTEM_SERVLET_RESOURCE_KEY = "File System";
   public static final String STREAM_CACHE_SERVLET_RESOURCE_KEY = "Stream Cache";
-  public static final String SERVER_CONFIGURATION_RESOURCE_KEY = "Server Configuration";
```

Comment:
I think this was added because there wasn't a way to retrieve the conf from the `FileSystem` until recently? Otherwise I have no clue why we used this approach.